manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.8.0
+PackageLocale: en-US
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
```

```suggestion
PackageName: OneScript engine
```
This must match Control Panel.